1-exercises/C-highlight-words/script.js

```diff
@@ -1,5 +1,33 @@
+let newPEl = document.createElement('p');
+let newSelectEl = document.createElement('SELECT');
+let content = document.getElementById('content')
+content.appendChild(newSelectEl)
+content.appendChild(newPEl)
+
+
 function highlightWords(paragraph, colours) {
```

Comment:
May be worth adding more than 1 function for this code, maybe 2 functions called createSelector and format paragraph.